sdk/eventhub/event-hubs/src/eventHubClient.ts

```diff
@@ -48,59 +48,66 @@ export interface RetryOptions {
 }
 
 /**
- * Options to passed when creating a producer using the EventHubClient.
+ * The set of options that can be specified when creating an `EventHubProducer`.
+ * These can be specified when creating the producer via the `createProducer` method.
  */
 export interface EventHubProducerOptions {
   /**
    * @property
-   * The id of the partition to which the event should be sent. If no id is provided,
-   * the service will determine the partition to which the event will be sent.
+   * The identifier of the partition that the producer will be bound to.
+   * If an id is provided, all events sent using the producer will reach the same partition.
+   * If no id is provided, the service will determine the partition to which the event will be sent.
    */
   partitionId?: string;
   /**
    * @property
-   * Retry options for the send operation on the producer. If no value is provided here, the
-   * retry options set when creating the `EventHubClient` is used.
+   * The retry options used to govern retry attempts when an issue is encountered while sending.
+   * If no value is provided here, the retry options set when creating the `EventHubClient` is used.
    */
   retryOptions?: RetryOptions;
 }
 
 /**
- * Options that can be passed when sending a batch of events using the EventHubsClient.
+ * The set of options that can be specified to influence the way in which events are sent to the
+ * Event Hubs service.
  */
 export interface SendOptions {
   /**
    * @property
-   * If specified, Event Hubs will hash this string to map to a partitionId.
+   * The partition hashing key to associate with the event or batch of events.
```

Comment:
> The partition hashing key used by the Event Hubs service to assign a partition to the event/events being sent

?

Reply:
Changing to
> A value that is hashed to produce a partition assignment.

Taken from [here](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-programming-guide#partition-key)